### PR TITLE
update to opencti 6.4.4

### PR DESCRIPTION
**Pull request by axelfahy**
- Update OpenCTI to 6.4.4
- Update redis to 7.4.1 in the docker-compose
- Update rabbitmq to 4.0 in the docker-compose

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.1] - 2024-12-18
```
Comment (debelyoo) [inline CHANGELOG.md:8]: the minor version should be bumped (not the patch)

Comment (axelfahy) [reply CHANGELOG.md]: 👍🏻 changed

File: entity/opencti_campaign.go [hunk 2]
```diff
@@ -64,7 +65,8 @@ func (c Campaign) CreateResponseField() string { return "campaignAdd" }
 
 // CampaignAddInput represents a GraphQL INPUT_OBJECT
 // Some fields from the OpenCTI schema may be missing
-// (See the examples for ways to expand an existing type).
+// (See the examples for ways to expand an existing type)
+//
```
Comment (TheWanderingKobold) [inline entity/opencti_campaign.go:69]: Please run `make fmt` again, there seems to be some unexpected format changes.

File: tools/gocti_type_generator/pyproject.toml [hunk 3]
```diff
@@ -13,7 +13,7 @@ license = {text  = "Apache 2.0"}
 version = "0.1.0"
```
Comment (TheWanderingKobold) [inline tools/gocti_type_generator/pyproject.toml:13]: GoCTI version should be updated here as well.

Comment (axelfahy) [reply tools/gocti_type_generator/pyproject.toml]: Thank you, I specified in the README that it should be updated here as well.

Comment (TheWanderingKobold) [reply tools/gocti_type_generator/pyproject.toml]: Good point

**Review by TheWanderingKobold — APPROVED** [review]
LGTM

**Review by debelyoo — APPROVED** [review]

**Pull request merged**